companion/src/firmwares/opentx/opentxinterface.cpp

```diff
@@ -1290,6 +1292,7 @@ void registerOpenTxFirmwares()
   openTx->addOption("haptic", QObject::tr("Haptic module installed"));
   openTx->addOption("lua", QObject::tr("Support for Lua model scripts"));
   openTx->addOption("nojoystick", QObject::tr("No Joystick emulation inside the FW (only Mass Storage as in the Bootloader)"));
+  openTx->addOption("mixersmon", QObject::tr("Adds mixers output view to the CHANNELS MONITOR screen, pressing [ENT] switches between the views"));
```

Comment:
Ok. I just need to think to update the compilation server which needs to support it.